src/lager_transform.erl

```diff
@@ -200,15 +200,15 @@ do_transform(Line, SinkName, Severity, Arguments0, Safety) ->
                   [],
                   %% trick the linter into avoiding a 'term constructed but not used' error:
                   %% (fun() -> {error, lager_not_running} end)()
-                  [{call, Line, {'fun', Line, {clauses, [{clause, Line, [],[], [{tuple, Line, [{atom, Line, error},{atom, Line, lager_not_running}]}]}]}}, []}]
+                  [{call, Line, {'fun', Line, {clauses, [{clause, Line, [],[], [{tuple, erl_anno:set_generated(true, Line), [{atom, Line, error},{atom, Line, lager_not_running}]}]}]}}, []}]
```

Comment:
You should no longer need the `fun`. As far as I understand, its only purpose was to suppress the warning. The call to `erl_anno:set_generated/2` is sufficient to suppress the warning.

The following should suffice:

```
[{tuple, erl_anno:set_generated(true, Line), [{atom, Line, error},{atom, Line, lager_not_running}]}]
```

Also remember to change the comment line above.

Reply:
Sure. Thanks for checking this. I went the lazy route, it seems, but glad you had a change to review.